gemfiles/ruby_2.7.3_cucumber3.gemfile.lock

```diff
@@ -67,7 +67,7 @@ GEM
     hashdiff (1.0.1)
     json-schema (2.8.1)
       addressable (>= 2.4)
-    libddwaf (1.3.0.0.0.beta1-x86_64-linux)
+    libddwaf (1.3.0.0.0.beta1)
```

Comment:
I looks harmless, and CI is happy for now.
I'm not 100% sure how to avoid this with our current Rake task we use to update dependencies, but if it's an issue we'll circle back to this.